cdap-ui/app/cdap/components/Experiments/CreateView/MLAlgorithmSelection/HyperParamWidget/index.js

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright © 2018 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+*/
+
+import PropTypes from 'prop-types';
+import React from 'react';
+require('./HyperParamWidget.scss');
+const SelectWidget = ({options, value, onChange}) => {
```

Comment:
Seems like you can just use `SelectWithOptions` component that we have, instead of creating this widget? From the code it seems like they do the same thing

Reply:
Good catch! 👍 Have updated it.